variants/circuitplay/variant.h

```diff
@@ -51,7 +51,10 @@
 #define portInputRegister(port)  (&(port->IN.reg))
 #define portModeRegister(port)   (&(port->DIR.reg))
 #define digitalPinHasPWM(P)      (g_APinDescription[P].ulPWMChannel != NOT_ON_PWM || g_APinDescription[P].ulTCChannel != NOT_ON_TIMER)
-#define digitalPinToInterrupt(P) (g_APinDescription[P].ulExtInt)
+
+#if (ARDUINO_SAMD_VARIANT_COMPLIANCE < 10606)
```

Comment:
hmm, can I ask why would we need this.

Reply:
From the commit message:

Starting in SAMD CORE 1.6.6, `digitalPinToInterrupt`
was moved to Arduino.h, variant.h must no longer define it.

Reply:
ah right, I think we could delete this altogether, the header did
```
#define ARDUINO_SAMD_VARIANT_COMPLIANCE 10610
```

So it is effectively a delete already. There is no need to keep it.